README.md

```diff
@@ -72,6 +76,8 @@ If you're ready to start contributing:
 * Airtable
 * Bottleneck
 * Heroku - Deployment pipeline
+* Jest testing
+* TravisCI
 * [Testing framework to come]
```

Comment:
I think writing Jest tests with Travis for CI is a suitable testing framework =D